tikv/deploy-tikv-using-ansible.md

```diff
@@ -0,0 +1,330 @@
+
+## Step 1: Download TiDB-Ansible to the Control Machine
+
+1. Log in to the Control Machine using the `tidb` user account and enter the `/home/tidb` directory.
+
+2. Download the corresponding TiDB-Ansible version. The default folder name is `tidb-ansible`.
+
+    - Download the 2.0 GA version:
+
+        ```bash
+        git clone -b release-2.0 https://github.com/pingcap/tidb-ansible.git
+        ```
+    
+    - Download the master version:
+
+        ```bash
+        git clone https://github.com/pingcap/tidb-ansible.git
+        ```
+
+    You can turn to the official team for advice on which version to choose.
+
+## Step 2: Install Ansible and dependencies in the Control Machine
+
+1. Install Ansible and dependencies on the Control Machine:
+
+    ```bash
+    sudo yum -y install epel-release
+    sudo yum -y install python-pip curl
+    cd tidb-ansible
+    sudo pip install -r ./requirements.txt
+    ```
+
+    Ansible and related dependencies are recorded in the `tidb-ansible/requirements.txt` file.
+
+2. View the version of Ansible:
+
+    ```bash
+    ansible --version
+    ```
+
+    Currently, the 1.0 GA version depends on Ansible 2.4, while the 2.0 GA version and the master version are compatible with Ansible 2.4 and Ansible 2.5.
+
+## Step 3: Edit the `inventory.ini` file to orchestrate the TiKV cluster
+
+Edit the `tidb-ansible/inventory.ini` file to orchestrate the TiKV cluster. This section shows the cluster topology of a single TiKV instance on a single machine and the cluster topology of multiple TiKV instances on a single machine.
+
+> **Note:** Leave `[tidb_servers]` in the `inventory.ini` file empty, because this deployment is for the TiKV cluster, not the TiDB cluster.
+
+The standard TiKV cluster contains 6 machines: 3 PD nodes and 3 TiKV nodes.
+
+- Deploy at least 3 instances for TiKV.
+- Do not deploy TiKV together with PD on the same machine.
+- Use the first PD machine as the monitoring machine.
+
+It is required to use the internal IP address to deploy. For more details, see [Software and Hardware Requirements](../op-guide/recommendation.md).
+
+### Option 1: Use the cluster topology of a single TiKV instance on each TiKV node
+
+| Name  | Host IP     | Services |
+|-------|-------------|----------|
+| node1 | 172.16.10.1 | PD1      |
+| node2 | 172.16.10.2 | PD2      |
+| node3 | 172.16.10.3 | PD3      |
+| node4 | 172.16.10.4 | TiKV1    |
+| node5 | 172.16.10.5 | TiKV2    |
+| node6 | 172.16.10.6 | TiKV3    |
+
+Edit the `inventory.ini` file as follows:
+
+```ini
+[tidb_servers]
+
+[pd_servers]
+172.16.10.1
+172.16.10.2
+172.16.10.3
+
+[tikv_servers]
+172.16.10.4
+172.16.10.5
+172.16.10.6
+
+[monitoring_servers]
+172.16.10.1
+
+[grafana_servers]
+172.16.10.1
+
+[monitored_servers]
+172.16.10.1
+172.16.10.2
+172.16.10.3
+172.16.10.4
+172.16.10.5
+172.16.10.6
+```
+
+### Option 2: Use the cluster topology of multiple TiKV instances on each TiKV node
```

Comment:
Why is this a recommended approach? What makes it better than the first? In what situations is it better?

Reply:
I'll add the corresponding content later. Thanks for all your comments!

Reply:
Okay! =D